tests/ut/xrd-control-plane/cp_configmap.bats

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bats
+
```

Comment:
Could you copy https://github.com/ios-xr/xrd-ha-app/blob/main/.github/workflows/chart-tests.yml to `.github/workflows/tests.yml` and modify appropriately please?  (i.e., change "Start container" step to follow your tests/README.md; remove "Run integration tests"; add separate steps for "Run xrd-control-plane unit tests" and "Run xrd-vrouter unit tests")

This will ensure these tests are run every PR (including this one)